From 6f5e8133651df38d187c05bc27fd8fa25f0216da Mon Sep 17 00:00:00 2001
Message-Id: <6f5e8133651df38d187c05bc27fd8fa25f0216da.1659172941.git.stefan@agner.ch>
In-Reply-To: <2d3d8c091bc692999a1c519c1dc1570cc8be2358.1659172941.git.stefan@agner.ch>
References: <2d3d8c091bc692999a1c519c1dc1570cc8be2358.1659172941.git.stefan@agner.ch>
From: Stefan Agner <stefan@agner.ch>
Date: Wed, 27 Apr 2022 20:36:19 +0200
Subject: [PATCH 10/13] ARM: dts: bcm2711: yellow: use generic activity trigger
 for green LED

Use the generic trigger "activity" for the green LED so that any
system activity is shown.

Signed-off-by: Stefan Agner <stefan@agner.ch>
---
 arch/arm/boot/dts/bcm2711-rpi-cm4-ha-yellow.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/bcm2711-rpi-cm4-ha-yellow.dts b/arch/arm/boot/dts/bcm2711-rpi-cm4-ha-yellow.dts
index 711a09441ad0..f2f4f1a49592 100644
--- a/arch/arm/boot/dts/bcm2711-rpi-cm4-ha-yellow.dts
+++ b/arch/arm/boot/dts/bcm2711-rpi-cm4-ha-yellow.dts
@@ -622,7 +622,7 @@ audio_pins: audio_pins {
 &leds {
 	act_led: led-act {
 		label = "led0";
-		linux,default-trigger = "mmc0";
+		linux,default-trigger = "activity";
 		gpios = <&gpio 42 GPIO_ACTIVE_HIGH>;
 	};
 
-- 
2.37.1

